common/src/main/resources/messages/messages.properties

```diff
@@ -138,6 +139,7 @@ cluster.scaling.stopstart.downscale.starting.identifiedrecoverycandidates=Scalin
 cluster.scaling.stopstart.downscale.couldnotdecommission=Could not decommission services on all nodes. Some nodes may need attention: instanceCount={0}, hostname(s): [{1}]
 cluster.scaling.stopstart.downscale.enteringcmmaintmode=Putting {0} cm-host(s) into maintenance mode
 cluster.scaling.stopstart.downscale.enteredcmmaintmode=Finished putting {0} cm-host(s) into maintenance mode
+cluster.scaling.stopstart.downscale.exclude.lost.nodes=Found {0} disconnected host(s): [{1}]. Excluding them from the scaling operation.
```

Comment:
should we say scaling or downscaling?

Reply:
Yes, that makes more sense. Updated these messages